<launch>
	<node pkg="px4_command" type="payload_drop" name="payload_drop" output="screen">

	<rosparam command="load" file="$(find px4_command)/config/payload_drop.yaml" />

	</node>
</launch>
